Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/pcr/PCRProcessor.kt

```diff
@@ -82,7 +82,7 @@ class PCRProcessor @Inject constructor(
         analyticsKeySubmissionCollector.reportTestRegistered()
 
         if (testResult == PCR_OR_RAT_PENDING) {
-            testResultScheduler.setPeriodicTestPolling(enabled = true)
+            testResultScheduler.setPcrPeriodicTestPollingEnabled(enabled = true)
```

Comment:
We can call `setRatResultPeriodicPollingMode` in the createMethod in `RapidAntigenProcessor`, analog to how the PCR polling is started.

Reply:
done!